### PR TITLE
Tweaks to Node.js build an app tutorial

### PR DESCRIPTION
**Pull request by jseldess**
- Comment edit in node txn wrapper
- Minor updates to flow of node txn step

### CODE REVIEW

File: build-a-nodejs-app-with-cockroachdb.md [hunk 1]
```diff
@@ -43,18 +43,32 @@ Now that you have a database and a user, you'll run code to create a table and i
...
 <a href="https://raw.githubusercontent.com/cockroachdb/docs/master/_includes/app/basic-sample.js" download>download it directly</a>.
 
 ~~~ js
 {% include app/basic-sample.js %}
 ~~~
 
+Then run the code:
+
+~~~ shell
+$ node basic-sample.js
```
Comment (justinj) [inline build-a-nodejs-app-with-cockroachdb.md]: Have we instructed them at this point to call the file this? Does that matter?

File: build-a-nodejs-app-with-cockroachdb.md [hunk 2]
```diff
@@ -63,7 +77,21 @@ You can copy the code or
 {% include app/txn-sample.js %}
 ~~~
 
-After running the code, use the [built-in SQL client](use-the-built-in-sql-client.html) to verify that funds were transferred from one account to another:
+Then run the code:
+
+~~~ shell
+$ node txn-sample.js
```
Comment (justinj) [inline build-a-nodejs-app-with-cockroachdb.md]: same here? I don't actually feel strongly, just want to be sure this is intentional.

File: build-a-nodejs-app-with-cockroachdb.md (hunk 1 above)
Comment (jseldess) [inline build-a-nodejs-app-with-cockroachdb.md]: Good point. I've tweaked the language a bit to be clearer about this. PTAL.

File: build-a-nodejs-app-with-cockroachdb.md (hunk 2 above)
Comment (jseldess) [inline build-a-nodejs-app-with-cockroachdb.md]: Same as above. If you prefer this new approach, I'll also update the other build an app tutorials accordingly.

**Comment by justinj** [comment]
LGTM!

**Pull request merged**